handlers/jobs.go

```diff
@@ -21,7 +21,7 @@ import (
 )
 
 var (
-	trueVal  = true
+	// trueVal  = true
```

Comment:
Why comment this out? Also why is this there in the first place :sweat_smile: ?

Reply:
Comment this out so the go lint won't complaint about defined, but unused variable; comment, instead of delete - so it can potentially be used later, if needed.

But it needed in the first place, because [`SecurityContext`](https://pkg.go.dev/k8s.io/api/core/v1#SecurityContext) needs `*bool`, which requires variable.